docs/ism/managedindices.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Managed Indices
+nav_order: 3
+parent: Index State Management
+has_children: false
+---
+
+# Managed Indices
+
+You can automate the changing or updating of a policy on a schedule using the managed index API.
+
+This table lists the parameters you can update using the managed index API:
+
+Parameter | Description | Type | Required | Read Only
+:--- | :--- |:--- |:--- |
+`name` |  The name of the managed index. | `string` | Yes | No
+`index` | The name of the index being managed. | `string` | Yes | No
+`index_uuid`  |  The uuid of the index being managed. | `string` | Yes | No
+`enabled` |  When this is true the managed index is scheduled and run by the scheduler. | `boolean` | Yes | No
```

Comment:
"When true, the managed index..."

Reply:
Lots of passive voice here, some of which might be OK, but I suspect there's a better way to word most of it.